drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-backend/src/main/java/org/drools/workbench/screens/scenariosimulation/backend/server/AbstractKieContainerService.java

```diff
@@ -32,12 +35,22 @@
     private BuildInfoService buildInfoService;
 
     protected KieContainer getKieContainer(Path path) {
-        KieModule kieModule = moduleService.resolveModule(path);
-        KieContainer kieContainer = buildInfoService.getBuildInfo(kieModule).getKieContainer();
-        if (kieContainer == null) {
-            throw new IllegalArgumentException("Retrieving KieContainer has failed. Fix all compilation errors within the " +
-                                                       "project and build the project again.");
+        final KieModule kieModule = moduleService.resolveModule(path);
```

Comment:
@Rikkola It could have sense to put here a comment explaining why we need to clone it? eg. Cloning is required to guarantee synchronization of the Buider info? WDYT?